### ISSUE TITLE
FlatUIUtils.nonUIResource() non copy all attributes

### ISSUE THREAD

**Issue by basix86** [JFormDesigner/FlatLaf#75]
Hi, I have see that FlatUIUtils.nonUIResource() create a new Font object that can be rendered differently from the original font.

I have see this bug when I have set the InternalFrame.titleFont with two different fonts object (extend and not extend UIResource).

The problem is the FlatUIUtils:124
`new Font( font.getName(), font.getStyle(), font.getSize() )`
This not copy all attributes.

To solve this bug, we can emulate copy constructor of FontUIResource that internally use a copy constructor of font.

In that case the problem is that the copy constructor of font is private.
To solve this limitation we can create another class 

```
	private static class FontNotUiResource extends Font {
		protected FontNotUiResource( Font font ) {
			super( font );
		}
	}
```

**Comment by DevCharly** [comment]
Thanks for reporting. This is now fixed in master branch.

Found a solution that works without another class (`font.deriveFont()`).

**Comment by DevCharly** [comment]
released in 0.28
